webpack.config.js

```diff
@@ -1,4 +1,5 @@
 
+
 module.exports = {
   entry: __dirname + '/client/dist/src/index.jsx',
```

Comment:
use path.join here, another thing eslint will yell at you for

Reply:
done!